### PR TITLE
Fix #128

### LINKED ISSUE #128: Little mistake in ru/part12
12.1. QUIC

Протокол QUIC (Quick UDP Internet Connections, Быстрые UDP Интернет Соединений) Гугла – это интересный эксперимент, проведённый в том же стиле, как и с SPDY. QUIC – это замена TCP + TLS + HTTP/2, реализованная поверх UDP.

"Быстрые UDP Интернет Соединений" replace to "Быстрые UDP интернет соединения"

### PR DESCRIPTION
**Pull request by d3th**

### CODE REVIEW

File: ru/part12.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ http2 по-прежнему имеет многое из устаревшего
 ## 12.1. QUIC
 
 Протокол [QUIC](https://www.chromium.org/quic) (Quick UDP Internet Connections,
-Быстрые UDP Интернет Соединений) Гугла – это интересный эксперимент,
+Быстрые UDP интернет соединения) Гугла – это интересный эксперимент,
```
Comment (olegstepura) [inline ru/part12.md]: I would write "интернет-соединения" (note tha dash), see also [explanation on why letters should be lower-case, and why dash should be there](http://rus.stackexchange.com/a/180)

**Comment by olegstepura** [comment]
Now it is ok :+1:

**Comment by vlet** [comment]
Ok :+1:

**Comment by bagder** [comment]
thanks!

**Pull request merged**